### PR TITLE
VALI-7683 Improve PRs sync description

### PR DESCRIPTION
**Pull request by ricardo-martins-altium**

### CODE REVIEW

File: sync-branch/action.yml [hunk 1]
```diff
@@ -60,3 +60,4 @@ runs:
       TO_BRANCH: ${{ inputs.to-branch }}
       REVIEWERS: '["${{ inputs.pr-author-id }}"]'
       PULL_REQUEST_AUTO_MERGE_METHOD: 'merge'
+      PULL_REQUEST_BODY: 'New code from {FROM_BRANCH} so let's bring {TO_BRANCH} up to speed! Please review and comment /merge to close the PR with the standard merge instead of squash'
```
Comment (rui-coelho-altium) [inline sync-branch/action.yml:63]: This change to the message doesn't seem okay:
`{FROM_BRANCH}` and `{TO_BRANCH}` don't exist in the context of the message. When the string interpolation occurs, an error will be thrown.

In this context, these variables are not available in the message, they are only inputs for the `tretuna/sync-branches@main` action.

For `{FROM_BRANCH}`, you should use `sync-${{ inputs.temp-branch }}`.
For `{TO_BRANCH}`, you should use `${{ inputs.to-branch }}`.

**Review by rui-coelho-altium — CHANGES_REQUESTED** [review]
Comments on line 63, file `sync-branch/action.yml`

File: sync-branch/action.yml (hunk 1 above)
Comment (filipe-galo-altium) [reply sync-branch/action.yml:63]: what rui said 👍

**Review by rui-coelho-altium — APPROVED** [review]
LGTM

**Review by filipe-galo-altium — APPROVED** [review]

**Pull request merged**